src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -78,7 +79,7 @@ function BaseHTMLEngineProvider(props) {
             baseStyle={styles.webViewStyles.baseFontStyle}
             tagsStyles={styles.webViewStyles.tagStyles}
             enableCSSInlineProcessing={false}
-            systemFonts={_.values(singleFontFamily)}
+            systemFonts={_.values(StyleUtils.fontFamily.singleFontFamily)}
```

Comment:
https://github.com/Expensify/App/issues/33036
Should we leave these imports for now instead? What are your thoughts on this idea

Reply:
done! 👍